releases/lib/common/router.dart

```diff
@@ -0,0 +1,111 @@
+// Copyright 2022 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io';
+
+import 'package:meta/meta.dart';
+import 'package:shelf/shelf_io.dart';
+
+import '../service/log.dart';
+import 'handler.dart';
+
+class Router {
```

Comment:
@ricardoamador I think most of the common source code would be applicable to a microservice package. I tried to make it light weight to set up the server based off how some of the golang server frameworks I've used.